src/antsibull_core/venv.py

```diff
@@ -102,4 +102,5 @@ def install_package(package_name: str) -> sh.RunningCommand:
             directly to :command:`pip install`.
         :returns: An :sh:obj:`sh.RunningCommand` for the pip output.
         """
+        # pylint:disable-next=broad-exception-raised
         raise Exception('Not implemented')
```

Comment:
This should raise a NotImplementedError

Reply:
That's not a change that should happen in this PR.